sdk/python/requirements/py3.9-requirements.txt

```diff
@@ -144,10 +144,12 @@ six==1.16.0
     # via python-dateutil
 sniffio==1.3.1
     # via anyio
-sqlalchemy[mypy]==2.0.29
+sqlalchemy[mypy]==1.4.52
```

Comment:
not saying it's wrong, more curious, can you please share what's causing the downgrade of sqlalchemy?

Reply:
I have just re-ran the command to regenerate this file but not sure why this was modified. I will pay attention to it.